_rules/iframe-accessible-name-cae760.md

```diff
@@ -150,12 +151,22 @@ Does not apply to non `iframe` element.
 
 #### Inapplicable Example 2
 
-`iframe` is not [included in the accessibility tree][].
+This `iframe` is not part of [sequential focus navigation][], nor [included in the accessibility tree][] because of `display: none;`.
 
 ```html
 <iframe style="display:none;" src="/test-assets/SC4-1-2-frame-doc.html"> </iframe>
 ```
 
+#### Inapplicable Example 3
+
+This `iframe` is not part of [sequential focus navigation][] because it has `tabindex="-1"` and not [included in the accessibility tree][] because of `role="presentation"`
```

Comment:
```suggestion
This `iframe` is neither part of [sequential focus navigation][] (because it has `tabindex="-1"`) nor ([included in the accessibility tree][] because of `role="presentation"`).
```

Reply:
I don't know what this is for, seems unnecessary.